modules/account-lib/package.json

```diff
@@ -43,6 +43,7 @@
     "@bitgo/bls-dkg": "^1.0.2",
     "@bitgo/statics": "^6.17.0",
     "@bitgo/sdk-core": "^1.0.1",
+    "@noble/secp256k1": "git+https://github.com/brandonblack/noble-secp256k1.git#856129c06436fb696bab7d827edad5effdffaee2",
```

Comment:
let's fork this into the bitgo org instead of using @brandonblack's fork. We can do this in a follow up.

Reply:
I have created a ticket BG-48681 for this.